src/cmd/singularity/cli/sif.go

```diff
@@ -0,0 +1,250 @@
+)
+
+// SifCmd represent the sif CLI cmd
+var SifCmd = &cobra.Command{
+	Use:  "sif",
+	Args: cobra.MinimumNArgs(1),
+	Run:  nil,
+}
+
+// SifCreate sif create cmd
+var SifCreate = &cobra.Command{
+	Use:     "create [option] <file>",
+	Short:   "Create a new sif file with input data objects",
+	Example: sifCreateExample,
+	Args:    cobra.MinimumNArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		var argc []string
+		argc = append(argc, "create")
+		if partfile != "" {
+			argc = append(argc, []string{"-P", partfile}...)
+		}
+		if fstype != "" {
+			argc = append(argc, []string{"-f", fstype}...)
+		}
+		if parttype != "" {
+			argc = append(argc, []string{"-p", parttype}...)
+		}
+		if content != "" {
+			argc = append(argc, []string{"-c", content}...)
+		}
+		if uuID != "" {
+			argc = append(argc, []string{"-c", uuID}...)
+		}
+		argc = append(argc, args...)
+		SifCmd := exec.Command(sif, argc...)
+		SifCmd.Stdout = os.Stdout
+		SifCmd.Stderr = os.Stderr
+
+		if err := SifCmd.Start(); err != nil {
+			sylog.Fatalf("%v", err)
+		}
+		if err := SifCmd.Wait(); err != nil {
+			sylog.Fatalf("%v", err)
+		}
+	},
+}
+
+// SifList sif list subcommand
+var SifList = &cobra.Command{
+	Use:     "list <file>",
+	Short:   "List SIF data descriptors from an input SIF file",
+	Example: sifListExample,
+	Args:    cobra.MinimumNArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+
+		argc := []string{"list", args[0]}
+		SifCmd := exec.Command(sif, argc...)
+		SifCmd.Stdout = os.Stdout
+		SifCmd.Stderr = os.Stderr
+
+		if err := SifCmd.Start(); err != nil {
+			sylog.Fatalf("%v", err)
+		}
+		if err := SifCmd.Wait(); err != nil {
+			sylog.Fatalf("%v", err)
+		}
+	},
+}
+
+// SifInfo sif info subcommand
+var SifInfo = &cobra.Command{
+	Use:     "info [id] <file>",
+	Short:   "Print data object descriptor info",
+	Example: sifInfoExample,
+	Args:    cobra.MinimumNArgs(2),
+	Run: func(cmd *cobra.Command, args []string) {
+
+		argc := append([]string{"info"}, args...)
+		SifCmd := exec.Command(sif, argc...)
+		SifCmd.Stdout = os.Stdout
+		SifCmd.Stderr = os.Stderr
+
+		if err := SifCmd.Start(); err != nil {
+			sylog.Fatalf("%v", err)
+		}
+		if err := SifCmd.Wait(); err != nil {
+			sylog.Fatalf("%v", err)
+		}
+	},
+}
+
+// SifDump sif dump subcommand
+var SifDump = &cobra.Command{
+	Use:     "dump [id] <file>",
+	Short:   "Display data object content",
+	Example: sifInfoExample,
+	Args:    cobra.MinimumNArgs(2),
+	Run: func(cmd *cobra.Command, args []string) {
+
+		argc := append([]string{"info"}, args...)
+		SifCmd := exec.Command(sif, argc...)
+		SifCmd.Stdout = os.Stdout
+		SifCmd.Stderr = os.Stderr
+
+		if err := SifCmd.Start(); err != nil {
+			sylog.Fatalf("%v", err)
+		}
+		if err := SifCmd.Wait(); err != nil {
+			sylog.Fatalf("%v", err)
+		}
+	},
+}
+
+// SifDel sif del subcommand
+var SifDel = &cobra.Command{
+	Use:     "del [id] <file>",
+	Short:   "Delete a specified set of descriptor+object",
+	Example: sifInfoExample,
```

Comment:
The example says to expect the same output as `singularity sif info`, but this command has no output. Remove `Example`.